NSS-Day-Cohort-66/rare-client-golden-droids-1
# Added Tag List view

I have the initial JSX structure, fetch call, and classes defined for the tag view.

## Steps to Test
1. Pull the `ts-AllTagsList` branch and switch to it.
2. Be sure to pull down `ts-tag-list` on the server side
3. Click on Tag Manager on navbar should take you to `http://localhost:3000/tags`

### Login as Admin (is_staff = true)
1. If user is an admin you should see a gear icon and a trash icon next to each tag
2. Verify all the tags from the database appear on page

### Login as Author (is_staff = false)
1. If user is an author you should just see a list of tags
2. Verify all the tags from the database appear on page

closes #13
